File: java/Disk.java
package DPBO.TP.TP3.java;

public class Disk {
    private String NameDisk;
    private int CapacityDisk;
    private int PriceDisk;

    public Disk() {
    }

    public Disk(String Name, int Capacity, int Price) {
        this.NameDisk = Name;
        this.CapacityDisk = Capacity;
        this.PriceDisk = Price;
    }

    public void setNameDisk(String Name) {
        this.NameDisk = Name;
    }

    public String getNameDisk() {
        return this.NameDisk;
    }

    public void setCapacityDisk(int Capacity){
        this.CapacityDisk = Capacity;
    }

    public int getCapacityDisk(){
        return this.CapacityDisk;
    }

    public void setPriceDisk(int Price) {
        this.PriceDisk = Price;
    }

    public int getPriceDisk() {
        return this.PriceDisk;
    }

    public void print() {
        System.out.println("Disk :");
        System.out.println();
        System.out.println("Name :" + this.getNameDisk());
        System.out.println("Capacity (Gb) :" + this.getCapacityDisk());
        System.out.println("Price (Rp) :" + this.getPriceDisk());
    }

}

File: python/Disk.py
class Disk:
    __NameDisk = ""
    __CapacityDisk = 0
    __PriceDisk = 0

    def __init__(self, Name="", Capacity = 0, Price=0):
        self.__NameDisk = Name
        self.__CapacityDisk = Capacity
        self.__PriceDisk = Price

    def setNameDisk(self, Name):
        self.__NameDisk = Name

    def getNameDisk(self):
        return self.__NameDisk

    def setCapacityDisk(self, Capacity):
        self.__CapacityDisk = Capacity

    def getCapacityDisk(self):
        return self.__CapacityDisk
        
    def setPriceDisk(self, Price):
        self.__PriceDisk = Price

    def getPriceDisk(self):
        return self.__PriceDisk

    def print(self):
        print("Disk:")
        print("Name : " + str(self.getNameDisk()))
        print("Capacity (Gb) : " + str(self.getCapacityDisk()))
        print("Price (Rp) :" + str(self.getPriceDisk()))